CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 Changelog
 ----------
 
+v0.2.2
+----
```

Comment:
Did Anselm also point out that now we turn on omnistaging by default? Or am I misremembering?

Reply:
Added it.